extras/jsonrpc/daemon_types.go

```diff
@@ -471,6 +471,50 @@ type UTXOListResponse []struct {
 	Type          string `json:"type"`
 }
 
+type UTXOReleaseResponse *string
+
+type TransactionListResponse []struct {
```

Comment:
there's a lot of code reuse here. I think I can clean this up by reusing the structures we already have